src/MyShop.Api/Controllers/CategoryController.cs

```diff
@@ -51,8 +50,8 @@ public async Task<IActionResult> DeleteCategory(Guid categoryId)
     }
 
     [HttpGet("categories")]
-    public async Task<ActionResult<IEnumerable<CategoryDto>>> GetCategories( )
-    => Ok(await _categoryService.GetAllCategoryAsync());
+    public async Task<ActionResult<IEnumerable<CategoryDto>>> GetCategories([FromQuery] GetCategories queries)
```

Comment:
GetCategories queries po co to jest?

Reply:
queries wykonuje sie w ciele taska     
[HttpGet("categories")]
 public async Task<ActionResult<IEnumerable<CategoryDto>>> GetCategories([FromQuery] GetCategories **queries**)
        => Ok(await _getCategoriesHandler.HandleAsync(**queries**));

Reply:
a jak inaczej można zrobić? żeby tak samo działało